helm/camel-k/values.yaml

```diff
@@ -48,3 +48,9 @@ operator:
 
   serviceAccount:
     annotations:
+
+  ## Extra environment variables.
+  ## ref: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
+  extraEnv: []
```

Comment:
Why don't we call it normally "envs" instead?

Reply:
Because `extraEnv` is a conventional name for additional env variables parameter almost in every existing helm chart.